examples/nomad-consul-ami/nomad-consul-docker.json

```diff
@@ -0,0 +1,86 @@
+{
+  "min_packer_version": "0.12.0",
+  "variables": {
+    "aws_region": "us-east-1",
+    "nomad_version": "0.6.3",
+    "consul_module_version": "v0.0.3",
+    "consul_version": "0.9.3"
+  },
+  "builders": [
+    {
+      "ami_name": "nomad-consul-docker-ubuntu-{{isotime | clean_ami_name}}",
+      "ami_description": "An Ubuntu 16.04 AMI that has Nomad, Consul and Docker installed.",
+      "instance_type": "t2.micro",
+      "name": "ubuntu16-ami",
+      "region": "{{user `aws_region`}}",
+      "type": "amazon-ebs",
+      "source_ami_filter": {
+        "filters": {
+          "virtualization-type": "hvm",
+          "architecture": "x86_64",
+          "name": "*ubuntu-xenial-16.04-amd64-server-*",
+          "block-device-mapping.volume-type": "gp2",
+          "root-device-type": "ebs"
+        },
+        "owners": [
+          "099720109477"
+        ],
+        "most_recent": true
+      },
+      "ssh_username": "ubuntu"
+    },
+    {
+      "ami_name": "nomad-consul-docker-amazon-linux-{{isotime | clean_ami_name}}",
+      "ami_description": "An Amazon Linux AMI that has Nomad, Consul and Docker installed.",
+      "instance_type": "t2.micro",
+      "name": "amazon-linux-ami",
+      "region": "{{user `aws_region`}}",
+      "type": "amazon-ebs",
+      "source_ami_filter": {
+        "filters": {
+          "virtualization-type": "hvm",
+          "architecture": "x86_64",
+          "name": "*amzn-ami-hvm-*",
+          "block-device-mapping.volume-type": "gp2",
+          "root-device-type": "ebs"
+        },
+        "owners": [
+          "amazon"
+        ],
+        "most_recent": true
+      },
+      "ssh_username": "ec2-user"
+    }
+  ],
+  "provisioners": [
+    {
+      "type": "shell",
+      "script": "setup_ubuntu16.sh",
+      "only": [
+        "ubuntu16-ami"
+      ]
+    },
+    {
+      "type": "shell",
+      "script": "setup_amazon-linux.sh",
```

Comment:
Please use `{{template_dir}}` so the paths work, even if you're running Packer from a different folder.